src/App.vue

```diff
@@ -130,8 +130,8 @@ main {
 	display: flex;
 	flex-direction: column;
 	align-content: center;
-	position: sticky;
-	top: 0;
+	position: fixed;
+	top: 70px;
 }
```

Comment:
I'm having an issue where the last event from the widget is not shown entirely and the widget isn't scrollable 

![image](https://user-images.githubusercontent.com/29747887/169565919-cc3e1883-75fc-4d58-90a2-7abb6b45c8aa.png)

Reply:
Should be fixed in latest commit